package.json

```diff
@@ -24,6 +25,7 @@
     "web"
   ],
   "dependencies": {
+    "bower": "^1.4.1",
```

Comment:
Close, something like this since we need bower as well https://github.com/meenie/heroku-buildpack-nodejs-grunt-bower